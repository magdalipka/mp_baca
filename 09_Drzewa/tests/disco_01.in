1
38
CREATE PREORDER 5 51 Adam Nowak 33 Anna Kowalska 44 Marek Mickiewicz 43 Zofia Krzak 47 Ola Nowicka
POSTORDER
PREORDER
INORDER
DEQUEMIN
DEQUEMAX
POSTORDER
NEXT 23
NEXT 43
PREV 33
PREV 47
DELETE 50
ENQUE 29 Tomasz Kowalski
DELETE 33
DELETE 47
ENQUE 20 Jan Nowak
ENQUE 28 Jan Kowalski
PREV 29
NEXT 29
DELETE 20
DELETE 29
ENQUE 48 Tomasz Wolny
DELETE 33
POSTORDER
ENQUE 35 Andrzej Wolny
ENQUE 38 Andrzej Kowalski
PREV 38
NEXT 38
PREV 35
NEXT 35
ENQUE 29 Tomasz Nowak
DELETE 29
INORDER
CREATE POSTORDER 1 51 Adam Nowak
DELETE 51
CREATE POSTORDER 1 51 Adam Nowak
ENQUE 48 Tomasz Wolny
HEIGHT
