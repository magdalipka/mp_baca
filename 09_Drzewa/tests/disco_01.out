ZESTAW 1
POSTORDER: 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz, 33 - Anna Kowalska, 51 - Adam Nowak
PREORDER: 51 - Adam Nowak, 33 - Anna Kowalska, 44 - Marek Mickiewicz, 43 - Zofia Krzak, 47 - Ola Nowicka
INORDER: 33 - Anna Kowalska, 43 - Zofia Krzak, 44 - Marek Mickiewicz, 47 - Ola Nowicka, 51 - Adam Nowak
DEQUEMIN: 33 - Anna Kowalska
DEQUEMAX: 51 - Adam Nowak
POSTORDER: 43 - Zofia Krzak, 47 - Ola Nowicka, 44 - Marek Mickiewicz
NEXT 23: BRAK
NEXT 43: 44 - Marek Mickiewicz
PREV 33: BRAK
PREV 47: 44 - Marek Mickiewicz
DELETE 50: BRAK
DELETE 33: BRAK
PREV 29: 20 - Jan Nowak
NEXT 29: 43 - Zofia Krzak
DELETE 33: BRAK
POSTORDER: 29 - Jan Kowalski, 43 - Zofia Krzak, 48 - Tomasz Wolny, 44 - Marek Mickiewicz
PREV 38: 35 - Andrzej Wolny
NEXT 38: 43 - Zofia Krzak
PREV 35: 29 - Jan Kowalski
NEXT 35: 38 - Andrzej Kowalski
INORDER: 29 - Tomasz Nowak, 35 - Andrzej Wolny, 38 - Andrzej Kowalski, 43 - Zofia Krzak, 44 - Marek Mickiewicz, 48 - Tomasz Wolny
DELETE 51: BRAK
HEIGHT: 1